common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -490,6 +490,7 @@ public static CharSequence unescapeCsv(CharSequence value) {
                             break;
                         }
                         // double-quote appears without being enclosed with double-quotes
+                    // fall through
                     case LINE_FEED:
```

Comment:
Actually, it seems like you don't need it there if the case is fully empty. Just verified.